sw/airborne/modules/sensors/temp_adc.h

```diff
@@ -28,12 +28,6 @@
 
 #include "std.h"
 
-/// flag to enable sending every new measurement via telemetry
-bool temp_adc_sync_send;
```

Comment:
I think it's meant to allow enabling/disabling the reporting at runtime, which IMHO is quite useful.